action.yml

```
@@ -1,24 +1,39 @@
 name: S3 Cache Action
 description:
-    Allows to skip a job if it already succeeded for the same repo state. Uses S3 for caching.
+    Allows to skip a job if it already succeeded for the same repo tree hash. Uses S3 for caching.
 author: Pleo OSS
 inputs:
-    bucket_name:
+    bucket-name:
         description:
             Name of the S3 bucket to use for storing cache files. The job needs to have AWS
             credentials configured to allow read/write from this bucket.
         required: true
-    key_prefix:
+    key-prefix:
         description:
             Key prefix to add to the cache files key. By default the job ID is used. The full key
             used for the cache files is `cache/${repoOwner}/${repoName}/${keyPrefix}/${treeHash}`
         default: ${{ github.job }}
         required: false
+    aws-region:
+        description: AWS region for the S3 bucket used for storing hashes.
```

Let's be consistent in how we call this

```suggestion
        description: AWS region for the S3 bucket used for cache files.
```

---

Done!